packages/components/psammead-navigation/src/DropdownNavigation/index.jsx

```diff
@@ -115,15 +118,28 @@ const iconBorder = css`
   border: ${GEL_SPACING_HLF} solid ${C_WHITE};
 `;
 
+// The sideLength of the button should be
+//  line height + top padding + bottom padding
+const calculateButtonSide = lineHeight =>
+  lineHeight / 16 + NAV_BAR_TOP_BOTTOM_SPACING * 2;
+
 const MenuButton = styled.button`
-  width: ${MENU_ICON_SIDE_LENGTH};
-  height: ${MENU_ICON_SIDE_LENGTH};
   position: relative;
   padding: 0;
   margin: 0;
   border: 0;
   background-color: transparent;
   ${({ dir }) => (dir === 'ltr' ? `float: left;` : `float: right;`)}
+  ${({ script }) => {
```

Comment:
Since this code is in two places, can we create a function passing the groupX.lineheight and just call it in here?

```
const getSideLength = groupLineHeight => {
  const sideLength = `${calculateButtonSide(groupLineHeight)}rem;`;
  return `
    height: ${sideLength};
    width: ${sideLength};    
  `;
};
```

```
 ${({ script }) => {
    return script && getSideLength(script.pica.groupA.lineHeight);
  }}
```